yt_dlp/extractor/facebook.py

```diff
@@ -524,6 +529,9 @@ def extract_metadata(webpage):
                     webpage, 'view count', default=None)),
                 'concurrent_view_count': get_first(post, (
                     ('video', (..., ..., 'attachments', ..., 'media')), 'liveViewerCount', {int_or_none})),
+                'like_count': post_stats.get('unified_reactors', {}).get('count'),
+                'comment_count': post_stats.get('total_comment_count'),
+                'repost_count': parse_count(post_stats.get('share_count_reduced')),
```

Comment:
Looking at the data the page returns, maybe `likers` seems like the better choice for `like_count`. Not sure if we really have a field that `unified_reactors` could map to.
We can also simplify and ensure the correct type is returned:
```suggestion
                **traverse_obj(post_stats, {
                    'like_count': ('likers', 'count', {int}),
                    'comment_count': ('total_comment_count', {int}),
                    'repost_count':('share_count_reduced', {parse_count}),
                }),
```